WooCommerce/Classes/ViewRelated/Orders/Order Creation/Synchronizer/OrderSynchronizer.swift

```diff
@@ -10,6 +10,11 @@ enum OrderSyncState {
     case error(Error, usesGiftCard: Bool)
 }
 
+enum OrderSyncBlockBehavior {
+    case allUpdates
+    case majorUpdates
```

Comment:
I find `majorUpdates` to be unclear in the sense of, _"What is exactly a major update, compared with any other update?"_. 

I'm not sure either if something like `.default` would be better thought, just because is the state we initialize the `RemoteOrderSynchronizer` in and it's the only other state it can be set to, or add additional cases just for readability?

```
enum OrderSyncBlockBehavior {
    case allUpdates
    case newLocalLineUpdates
    case bundleConfigurationUpdates
}
```